introduction.md

```diff
@@ -0,0 +1 @@
+Jeff Belina is a database developer taking the code school Mastering GitHub course.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.